index.php

```diff
@@ -0,0 +1,161 @@
+<?php
+	require __DIR__ . '/vendor/autoload.php';
+
+	use Twig\Environment;
+	use Twig\Loader\FilesystemLoader;
+	
+	$loader = new FilesystemLoader(__DIR__ . '/templates');
+	$twig = new Environment($loader);
+	
+	session_start();
+
+    ini_set('display_errors', 1);
+    error_reporting(E_ALL);
+	require __DIR__ . '/vendor/autoload.php';
+	define("IN_INDEX", 1);
+
+    include("config.inc.php");
+
+    if (isset($config) && is_array($config)) {
+
+        try {
+            $dbh = new PDO('mysql:host=' . $config['db_host'] . ';dbname=' . $config['db_name'] . ';charset=utf8mb4', $config['db_user'], $config['db_password']);
+            $dbh->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+        } catch (PDOException $e) {
+            print "Nie mozna polaczyc sie z baza danych: " . $e->getMessage();
+            exit();
+        }
+
+    } 
+	else {
+        exit("Nie znaleziono konfiguracji bazy danych.");
+    }
+
+    include("functions.inc.php");
+
+	$date = date('Y-m-d');
+	$userFeedback = "";
+	$ile_photos = 0;
+	$last_post = 0;
+	$comment = "";
+	$currentDate = date("Y-m-d H:i:s");
+
+	$posts = [];
+	$stmt = $dbh->prepare("SELECT p.*, u.login FROM a30_Post p INNER JOIN a30_Users u ON p.userID = u.id ORDER BY createdTime DESC LIMIT 72");
+	$stmt->execute();
+
+	while ($row = $stmt->fetch(PDO::FETCH_ASSOC)) {
+
+		$stmt_1 = $dbh->prepare("SELECT COUNT(*) FROM a30_User_Post_Like WHERE PostID = :PostID");
```

Comment:
More meaningful names like 'stmt_like_count' would be better than 'stmt_1' or 'stmt_2'.